powerhome/playbook
# [YNGJ-574] Background Kit - Set Background Size

#### Screens

<img width="1454" alt="Screen Shot 2022-03-22 at 2 19 18 PM" src="https://user-images.githubusercontent.com/2293844/159558860-c57b9e0b-37a0-4cae-a340-52aab2add39e.png">


#### Breaking Changes

No, this adds a new feature to existing kit, does not change existing features.

#### Runway Ticket URL

[[YNGJ-574]](https://nitro.powerhrg.com/runway/backlog_items/YNGJ-574)

#### How to test this

Go to playbook kit docs and view the size, it should show the style that has been set with cover as default.

added automated tests

#### Checklist:
